src/main/java/edu/cornell/kfs/sys/service/impl/CUMarshalServiceImpl.java

```diff
@@ -141,6 +156,22 @@ public <T> T unmarshalStream(InputStream inputStream, Class<T> clazz)
         Unmarshaller unmarshaller = createUnmarshaller(clazz, Optional.empty());
         return unmarshalStream(inputStream, unmarshaller);
     }
+    
+    @Override
+    public <T> T unmarshalStreamIgnoreDtd(InputStream inputStream, Class<T> clazz)
+            throws JAXBException, IOException, XMLStreamException {
+        JAXBContext jc = JAXBContext.newInstance(clazz);
+
+        XMLInputFactory inputFactory = XMLInputFactory.newFactory();
+        inputFactory.setProperty(XMLInputFactory.SUPPORT_DTD, false);
+        XMLStreamReader streamReader = inputFactory.createXMLStreamReader(inputStream);
+
+        Unmarshaller unmarshaller = jc.createUnmarshaller();
+        T response = (T) unmarshaller.unmarshal(streamReader);
+        CuXMLStreamUtils.closeQuietly(streamReader);
```

Comment:
This close-quietly call should be placed in a `finally` block.

Reply:
ok